django/core/management/utils.py

```diff
@@ -173,3 +174,64 @@ def run_formatters(written_files, black_path=(sentinel := object())):
             [black_path, "--fast", "--", *written_files],
             capture_output=True,
         )
+
+
+SPACE_PREFIX = "    "
+BRANCH_PREFIX = "|   "
+TEE_PREFIX = "|___ "
+ELBOW_PREFIX = "|___ "
```

Comment:
I think I would put these inside the function
I also see no difference (or very very little difference) between TEE and ELBOW